charts/memgraph/templates/statefulset.yaml

```diff
@@ -40,6 +40,11 @@ spec:
           image: "{{ $imageName }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           command: ["/usr/lib/memgraph/memgraph"]
+          args:
+          {{- range .Values.memgraphConfig }}
+          - {{ . | quote }}
+          {{- end }}
+
```

Comment:
i suppose you already tested it with template command

Reply:
I've been actively using Tilt for the import subjects, yes.